src/app/public/modules/file-attachment/file-drop.component.ts

```diff
@@ -30,33 +30,57 @@ import {
   styleUrls: ['./file-drop.component.scss']
```

Comment:
Not sure exactly where to note it, but the existing docs at https://developer.blackbaud.com/skyux/components/file-attachments/file-attachment#file-drop-properties have a couple lead-in paragraphs before the properties table for `SkyFileDropComponent`. Seems like we should port that content over to a JSDocomment for the component.

Reply:
I added that content under the `SkyFileDropComponent` section

Reply:
Looks like we copied over the second paragraph but not the first paragraph. Was that intentional?